### ISSUE TITLE
Move StopwatchExtension to pioneer

### ISSUE THREAD

**Issue by smoyer64** [junit-pioneer/junit-pioneer#38]
From the example code in the JUnit 5 documentation - https://github.com/selesy/junit5/blob/issues/343-document-testrule-compatibility/documentation/src/test/java/example/testrule/StopwatchExtension.java

**Comment by Bukama** [comment]
As I'm struggeling at the same as when I tried to implement the `@Issue` annotation I'll post a link to the branch here and if anyone want's to adapt the current state feel free to do so.

https://github.com/Bukama/junit-pioneer/tree/stopwatch

My problems are:

1. Entering the extension context inside a test to verify if the values of another store are beeing stored
2. Verifiy if the `TestReporter` prints the information correct inside the resulting XML


I checked via a `System.out` that the extension is executed, but I don't find a way to enter the context from a test class. I tried to pass it via n paramter, but this leads to an error, that there's no `ParameterResolver` for the `ExtensionContext` class.

I also tried to pass an `TestReporter` parameter. This can be resolved, but the interface only has methods to publish something via the reporter, but not to check if it contains the right data.

In both cases there are no further explanations in the Jupiter documentation.

Maybe you find a way to verify the execution of the extension.
